packages/xo-server/src/xapi/index.mjs

```diff
@@ -887,7 +887,8 @@ export default class Xapi extends XapiBase {
             } catch (error) {
               return `${hostNameLabel}: ${error.message}`
             }
-          })
+          }),
+          error.code
```

Comment:
Top!

Is it better than `error.message`? :thinking:

Reply:
IMHO,  yes and  `error.message` will be `NO_HOST_AVAILABLE()`

Reply:
Yes, but `error.message` will also contains any additional error params.

I think the best answer would be to use `error.message` as the message and copy both `code` and `params` of the original error to the aggregate one.